src/api/InkAPI.cc

```diff
@@ -6575,10 +6575,6 @@ tsapi::c::TSPluginVCIOBufferWaterMarkGet(TSHttpTxn txnp)
   return TS_IOBUFFER_WATER_MARK_PLUGIN_VC_DEFAULT;
 }
 
-// The API below require timer values as TSHRTime parameters
-// which are in nanoseconds. Use the TS_HRTIME macros defined
```

Comment:
Isn't it useful to know the parameters are in nanoseconds?

Reply:
No it’s in the docs